src/Message/SingleProductMessage.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Netflie\WhatsAppCloudApi\Message;
+
+final class SingleProductMessage extends Message
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected string $type = 'product';
+
+    private int $catalog_id;
+
+    private string $product_retailer_id;
+
+    private ?string $body;
+
+    private ?string $footer;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(string $to, int $catalog_id, string $product_retailer_id, ?string $body, ?string $footer, ?string $reply_to)
```

Comment:
`$body, $footer, $reply_to` maybe should are defined with a default value? `null`?